velox/common/base/Status.h

```diff
@@ -489,6 +490,10 @@ inline Status genericToStatus(Status&& st) {
 }
 
 } // namespace internal
+
+template <typename T>
```

Comment:
maybe add a comment block here explaining how we intend this to be used with a few examples?

Reply:
Updated. Thanks.